src/js/vue-plugins/document-utils.js

```diff
@@ -149,11 +149,11 @@ export default function install(Vue) {
 
         if (parent.type === 'o' && child.type === 'r') {
           this.propagateRoutePropertiesToOutingProperties(child, parent);
-        } else if (parent.type === 'r' && child.type === 'w') {
-          this.propagateGeomPoint(child, parent);
-        } else if (parent.type === 'x' && child.type === 'r') {
-          this.propagateGeomPoint(child, parent);
-        } else if (parent.type === 'x' && child.type === 'o') {
+        } else if (
+          (parent.type === 'r' && child.type === 'w') ||
+          (parent.type === 'x' && child.type === 'r') ||
+          (parent.type === 'x' && child.type === 'o')
```

Comment:
parentheses are useless because of operator priority. (unless prettier does a bad job at splitting?)

Reply:
Nope, prettier add them.